lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -751,11 +753,19 @@ spec = do
                     }
             in
                 x' === x .&&. show x' === show x
-        it "ApiByronWalletMigrationInfo" $ property $ \x ->
+        it "ApiWalletMigrationInfo" $ property $ \x ->
             let
-                x' = ApiByronWalletMigrationInfo
+                x' = ApiWalletMigrationInfo
                     { migrationCost =
-                        migrationCost (x :: ApiByronWalletMigrationInfo)
+                        migrationCost (x :: ApiWalletMigrationInfo)
+                    }
+            in
+                x' === x .&&. show x' === show x
+        it "ApiWalletMigrateData" $ property $ \x ->
+            let
+                x' = ApiWalletMigrateData
+                    { passphrase = passphrase (x :: ApiWalletMigrateData ('Testnet 0))
+                    , addresses = addresses (x :: ApiWalletMigrateData ('Testnet 0))
                     }
```

Comment:
```suggestion
                x' = ApiWalletMigrateData
                    { passphrase =
                        passphrase (x :: ApiWalletMigrateData ('Testnet 0))
                    , addresses =
                        addresses (x :: ApiWalletMigrateData ('Testnet 0))
                    }
```

Reply:
done in https://github.com/input-output-hk/cardano-wallet/pull/1664/commits/cf2b9e58c71fd7e80f8a9e1b9073348dfba3a67e